ament_cmake_test/cmake/run_test.py

```diff
@@ -87,6 +92,17 @@ def main(argv=sys.argv[1:]):
             if e.errno != errno.EEXIST:
                 raise
 
+    if args.skip_test:
+        # generate a skipped test result file
+        skipped_result_file = _generate_result(
+            args.result_file,
+            None,
+            True
```

Comment:
Keyword arguments should be passed with the keyword rather then as a positional argument. Then you also don't need the default value for `failure_message`.